src/main/frontend/src/components/JobItem.vue

```diff
@@ -27,11 +27,11 @@ const sortedSets = computed(() => props.sets.slice().sort())
                 </v-list-item>
                 <v-list-item density="compact" prepend-icon="mdi-archive">
                     <v-list v-if="isSelectiveHarvest">
-                        <span v-for="set in sortedSets" :key="set" density="compact">
-                            <a
-                                :href="`${repositoryBaseURL}?verb=ListRecords&set=${set}&metadataPrefix=${metadataPrefix}`">
-                                {{ set }} </a
-                            >,
+                        <span v-for="(set, index) in sortedSets" :key="set" density="compact">
+                            <a :href="`${repositoryBaseURL}?verb=ListRecords&set=${set}&metadataPrefix=${metadataPrefix}`">
```

Comment:
There's a GitHub Actions warning on this line; also, is the ":" before the "href" a typo, or have the updated the "<a>" spec since the last time I did HTML?

Reply:
Yes, it's a formatting check that doesn't show up with mvn checkstyle:check and difficult to see since we can't run prl-harvester locally right now. Just fixed it. Updated specs for HTML regarding the ":" before "href".